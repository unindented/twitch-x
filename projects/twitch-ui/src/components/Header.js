import React from 'react'
import './Header.css'

export default function Header () {
  return (
    <h1 className='Header'>
      Hello, world!
    </h1>
  )
}
