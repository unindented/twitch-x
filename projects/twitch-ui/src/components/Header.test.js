import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow'
import Header from './Header'

describe('Header', () => {
  let tree

  beforeEach(() => {
    const renderer = new ShallowRenderer()
    renderer.render(<Header />)
    tree = renderer.getRenderOutput()
  })

  it('renders', () => {
    expect(tree).toMatchSnapshot()
  })
})
