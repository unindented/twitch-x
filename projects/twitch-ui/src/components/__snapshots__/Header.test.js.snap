// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders 1`] = `
<h1
  className="Header"
>
  Hello, world!
</h1>
`;
